master/buildbot/test/unit/test_reporters_pushover.py

```diff
@@ -0,0 +1,93 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import sys
+from unittest import SkipTest
+
+from twisted.internet import defer
+from twisted.trial import unittest
+
+from buildbot.process.results import SUCCESS
+from buildbot.reporters.pushover import PushoverNotifier
+from buildbot.test.fake import httpclientservice as fakehttpclientservice
+from buildbot.test.fake import fakemaster
+from buildbot.test.util.config import ConfigErrorsMixin
+from buildbot.util import httpclientservice
+
+py_27 = sys.version_info[0] > 2 or (sys.version_info[0] == 2
```

Comment:
please remove this line. It looks like its not used. For further code, please use this instead:

    from future import PY3